docs/library/spi_engine/instruction-format.rst

```diff
@@ -197,9 +203,15 @@ bus behavior.
    * - Bits
      - Name
      - Description
-   * - [7:3]
+   * - [7:4]
      - reserved
      - Must always be 0.
+   * - [3]
+     - cs_active_high
+     - Configures the polarity of CS. When 0 (default), CS is active-low as 
+     - usual. If set to 1, CS will be active-high. Note that there's no need 
+     - to change the commands: for cs_active_high=1, the value of CS will be 
+     - automatically inverted with respect to the command.
```

Comment:
It is helpful to driver writers to include the version a feature was added. So suggest adding here:

This feature was added in vX.Y.Z. Setting this bit in older versions has no effect.